server/protocols/jmap/src/test/java/org/apache/james/jmap/methods/SetVacationResponseMethodTest.java

```diff
@@ -19,14 +19,210 @@
 
 package org.apache.james.jmap.methods;
 
-import org.apache.commons.lang.NotImplementedException;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
+
+import java.util.List;
+import java.util.Locale;
+import java.util.stream.Stream;
+
+import org.apache.james.jmap.api.vacation.AccountId;
+import org.apache.james.jmap.api.vacation.Vacation;
+import org.apache.james.jmap.api.vacation.VacationRepository;
+import org.apache.james.jmap.model.ClientId;
+import org.apache.james.jmap.model.GetMailboxesRequest;
+import org.apache.james.jmap.model.SetError;
+import org.apache.james.jmap.model.SetMailboxesRequest;
+import org.apache.james.jmap.model.SetVacationRequest;
+import org.apache.james.jmap.model.SetVacationResponse;
+import org.apache.james.jmap.model.VacationResponse;
+import org.apache.james.mailbox.MailboxSession;
+import org.junit.Before;
 import org.junit.Test;
 
+import com.google.common.collect.ImmutableMap;
+
 public class SetVacationResponseMethodTest {
 
-    @Test(expected = NotImplementedException.class)
-    public void setVacationResponseMethodIsNotImplemented() {
-        new SetVacationResponseMethod().process(null, null, null);
+    public static final String WRONG_ID = "WrongId";
+    public static final String TEXT_BODY = "Text body";
+    public static final String USERNAME = "username";
+    public static final MailboxSession.User USER = new MailboxSession.User() {
```

Comment:
You could define a TestSession once and avoid generating so many lines every time